stage0/app.go

```diff
@@ -357,7 +357,12 @@ func RmApp(dir string, uuid *types.UUID, usesOverlay bool, appName *types.ACName
 		}
 
 		if err := callEntrypoint(dir, appStopEntrypoint, args); err != nil {
-			return err
+			status, err := common.GetExitStatus(err)
+			// ignore nonexistent units failing to stop. Exit status 5
+			// comes from systemctl and means the unit doesn't exist
+			if err != nil || status != 5 {
```

Comment:
In `GetExitStatus`, if `err != nil` and `err.(*exec.ExitError)` succeeds, it means `status != 0` because "An ExitError reports an unsuccessful exit by a command.". If it doesn't succeed, we will `return -1, err`, and `err != nil` so the invariant will also be maintained.

Maybe I'm missing something? :/